modules/apps/web-experience/site-navigation/site-navigation-admin-web/src/main/resources/META-INF/resources/js/SiteNavigationMenuEditor.es.js

```diff
@@ -6,21 +6,25 @@ import position from 'metal-position';
 /**
  *	Site navigation menu editor component.
  */
+
```

Comment:
Hey @hhuijser, this extra line is likely going to break doc generation. Is there a way to avoid it?

/cc @mwilliams2014 can you double check to see if these files generate the documentation properly?

Reply:
Yeah, the build fails for me. When I remove the spaces, it still fails though with the same error, so I don't know that it's caused by that:
```
FAILURE: Build failed with an exception.

* Where:
Build file 'C:\Users\liferay\opt\git-stuff\liferay-portal\modules\apps\web-experience\site-navigation\site-navigation-admin-web\build.gradle' line: 17

* What went wrong:
A problem occurred evaluating project ':apps:web-experience:site-navigation:site-navigation-admin-web'.
> Project with path ':apps:foundation:frontend-taglib:frontend-taglib' could not be found in project ':apps:web-experience:site-navigation:site-navigation-admin-web'.
```

cc @Ithildir

Reply:
@mwilliams2014, remove the `settings.gradle` file?

Reply:
@Ithildir yep that solved the error 👍 the docs generate fine with the spaces @jbalsas, but for consistent formatting, they should probably be removed. Also, please make sure you add @review tags to all new jsdoc so I know to review it. Thanks!

Reply:
👍 @p2kmgcl will take care of that. As for formatting, whatever comes out of this would be made consistent across all other codebases. Still looks weird that they're separated by a blank line, I don't think that's the case for Java. @hhuijser?

Reply:
The existing jsdoc that has been sent for review does not have spaces. EX: https://github.com/liferay/liferay-portal/blob/master/modules/apps/foundation/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/liferay/app/App.es.js

Reply:
Whatever Hugo decides to do on this will become the standard and will be applied to the rest of already existing code.

Reply:
@mwilliams2014 @jbalsas yep, I will add missing `@review` tags :)

Reply:
Sorry, I am bit late with my response...
To me, it makes sense to get rid of that empty line. This is all enforced by Nate's tool.
I'll discuss with Nate what the proper workflow is for any issues/requests with formatting *.js

Reply:
@hhuijser @jbalsas I also noticed in some of our docs that we use @inheritdoc tags. This tag doesn't generate the parent class's jsdoc as you would expect, so we may have to rethink that, unless I'm missing something. Also, there are some cases where additional comments are used with @inheritdoc. These are erased when you use @inheritdoc. See https://github.com/liferay/liferay-portal/blob/616b7a5135daabe513eb69fdee328272ffaa9fb4/modules/apps/foundation/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/liferay/app/App.es.js#L189-L195 for example. How do we want to handle these?